docs/anomaly-detection/Presets.md

```diff
@@ -0,0 +1,136 @@
+---
+sort: 1
+weight: 1
+title: Presets
+menu:
+  docs:
+    parent: "anomaly-detection"
+    weight: 1
+    title: Presets
+aliases:
+- /anomaly-detection/Presets.html
+---
+# Presets
+> Check [Quickstart guide](https://docs.victoriametrics.com/anomaly-detection/quickstart/) to run `vmanomaly` using docker image.
+
+> Presets are available from v1.12.0
+
+Presets enable to detect anomalies in indicators that are hard to monitor using static alerts. 
+We use predefined machine learning models on different time frames that helps adjusting alerts to changing behavior.
```

Comment:
Part of this info is mentioned below. I deleted the part where we explain to the user what they don't need to specify. As if they don't need to specify, they don't need to read about it.
Combining all of the efforts it will conclude:
```
Presets enable anomaly detection in indicators that are hard to monitor using alerts based on static thresholds. 
So the anomaly detection alerting rules based on the [`anomaly_scores`](https://docs.victoriametrics.com/anomaly-detection/faq/#what-is-anomaly-score) stay the same over time, 
while we use predefined machine learning models for generating them.
Models are constantly retraining on different time frames which helps to keep alerts up to date and to consider underlying data patterns.

You can set up the simplified configuration file for `vmanomaly` just specifying the type of preset and data sources in [`reader`](https://docs.victoriametrics.com/anomaly-detection/components/reader/) and [`writer`](https://docs.victoriametrics.com/anomaly-detection/components/writer/) sections of the config.
The rest of the parameters are already set up for you.
```